### PR TITLE
Created stanjdev.json

### PR DESCRIPTION
**Pull request by stanjdev**
Starred the repo and added myself to the contributors list!

Thank you!

### CODE REVIEW

File: src/contributors/stanjdev.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Stan Jeong", 
+  "desc": "Front End Developer",
+  "github": "https://github.com/stanjdev",
+  "twitter": "",
```
Comment (iamdarshshah) [inline src/contributors/stanjdev.json:5]:
```suggestion
  
```
Don't push the `null` param. Remove it, if you don't have the link

**Comment by stanjdev** [comment]
working on it right now!

**Review by stanjdev — COMMENTED** [review]
Removed the twitter property, is this okay? @iamdarshshah

**Review by iamdarshshah — APPROVED** [review]
LGTM! Thanks @stanjdev

**Pull request merged**